swim/ping_request_test.go

```diff
@@ -172,7 +172,9 @@ func TestIndirectPing2(t *testing.T) {
 	cont := make(chan bool, 1)
 
 	tnodes := genChannelNodes(t, 4)
-	defer destroyNodes(tnodes...)
+	defer func() {
```

Comment:
A couple of lines below, there's another node added to the `tnode` slice. That extra node won't be destroyed without wrapping `destroyNodes` in another func.